Defect/de5503 mobile podcast alignment

zarahzachz opened the pull request:
Updating the image sizes on the podcast artwork and modifying the podcast jumbotron height to account for the new sizes.

Visit `/podcasts/ikr` to test.

This solution will show the appropriate padding for podcast art on mobile.

No corresponding PRs.

tcmacdonald commented on _assets/stylesheets/_podcast.scss:
```diff
@@ -39,28 +39,29 @@
     display: block;
 
-    @media screen and (min-width: $screen-sm) {
+    @media screen and (min-width: 375px) {
```
Is it necessary to hardcode `375px` here? Seems arbitrary.

zarahzachz commented:
@tcmacdonald I removed references to the 375px breakpoint. I'd originally used it to target iPhone X-wide screens and because the image got pretty large before the `screen-xs` breakpoint came up. But in retrospect, it isn't a big deal.

tcmacdonald commented on _assets/stylesheets/_podcast.scss:
```diff
@@ -39,28 +39,29 @@
+    &,
+    .podcast-background,
+    .header-content {
+      height: 730px;
```
Should this be `min-height`?

tcmacdonald commented on _assets/stylesheets/_podcast.scss:
```diff
@@ -69,55 +70,36 @@
+      &,
+      .podcast-background,
+      .header-content {
+        height: 315px;
```
Should this be `min-height`?

tcmacdonald commented on _assets/stylesheets/_podcast.scss:
```diff
@@ -69,55 +70,36 @@
+      &,
+      .podcast-background,
+      .header-content {
+        height: 350px;
```
Should this be `min-height`?

zarahzachz commented:
@tcmacdonald I had to rework this defect a bit because I didn't know we were supporting backgrounds with imgix. Summary of changes:
- Refactored podcast show header background to use background images
- Cleaned up some unnecessary markup (stuff like extra `row` classes)
- Removed the `min-height` solution I originally used. The header content should determine the height of the header and will grow/shrink accordingly.
- Refactored how the podcast artwork was being displayed so we get that "cut-out" effect in the design on desktop sized screens.

I checked these changes out using the devices listed in the responsive tabs in both Firefox and Chrome.

zarahzachz commented:
@tcmacdonald I forgot to account for the other imgix size. Updated CSS to work with it now.

Ready for review.
